package.json

```diff
@@ -23,7 +23,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "body-parser": "^1.19.0",
-    "clinical-trial-matching-service": "github:mcode/clinical-trial-matching-service",
+    "clinical-trial-matching-service": "github:mcode/clinical-trial-matching-service#add-address-to-location",
```

Comment:
Not totally sure about this but should this line be reverted back to just `"clinical-trial-matching-service": "github:mcode/clinical-trial-matching-service",`?

Reply:
Not until the pull request in clinical-trial-matching-service is merged - it needs the Address type that's part of the FHIR types.